client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/HashBasedShuffleWriter.java

```diff
@@ -353,6 +351,15 @@ protected void mergeData(int partitionId, byte[] buffer, int offset, int length)
     writeMetrics.incBytesWritten(bytesWritten);
   }
 
+  private void abort() throws IOException {
+    try {
+      dataPusher.waitOnTermination();
```

Comment:
stop immediately instead of wait on termination?

Reply:
It seems that `DataPusher` does not have a stop immediately method.